.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+All pull requests must have an associated issue in the issue tracker. If there
```

Comment:
Something like, you can read about our issues and PR processes here... and link the corresponding BEP.